### PR TITLE
feat(data-manage): edit pages for definitions

### PR DESCRIPTION
**Pull request by alexkim205**
## Changes

Implements edit pages for event and event property definition detail pages.

https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=10788%3A53084

https://user-images.githubusercontent.com/13460330/171679350-a4943afa-8c18-454a-b51e-ffb7dcb21027.mov

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

- [x] logic tests

### CODE REVIEW

**Comment by liyiy** [comment]
changing the name doesn't work 

https://user-images.githubusercontent.com/25164963/172186488-5aa4e28a-b436-4b59-9f9d-aa113aa23dea.mov

**Review by liyiy — DISMISSED** [review]
Are definition property pages supposed to have events? There's also a double border line here and the name editing also doesn't work

Screen Shot 2022-06-06 at 11 08 03 AM

File: frontend/src/scenes/data-management/definition/definitionEditLogic.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import { initKeaTests } from '~/test/init'
+import { definitionEditLogic } from 'scenes/data-management/definition/definitionEditLogic'
+import { expectLogic } from 'kea-test-utils'
+import { eventDefinitionsModel } from '~/models/eventDefinitionsModel'
+import { eventDefinitionsTableLogic } from 'scenes/data-management/events/eventDefinitionsTableLogic'
+import { eventPropertyDefinitionsTableLogic } from 'scenes/data-management/event-properties/eventPropertyDefinitionsTableLogic'
+import { router } from 'kea-router'
+import { urls } from 'scenes/urls'
+
+describe('definitionEditLogic', () => {
```
Comment (liyiy) [inline frontend/src/scenes/data-management/definition/definitionEditLogic.test.ts:13]: 👍 nice

**Comment by alexkim205** [comment]
Good catch, custom names for event definitions isn't available yet, so I'll make that input read only for now. Also going to hide it from property definitions as well.

> Are definition property pages supposed to have events?

Yup you should now be able to see the list of raw events that match the definition name. We had this in the previous version of definition drawers but it was removed when we removed detail views of definitions.

**Review by clarkus — APPROVED** [review]
Looks good to me. The only things I'd consider changing are those items that @alexkim205 brought up in [figma](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=11060%3A50972). 

For read only views, including some representation of "verified" status and the raw event name. We can borrow from the experiments feature header patterns to include a high-levle status pill but also an alternate name value for the item being described. 

data management - detail view

**Comment by alexkim205** [comment]
Thanks @liyiy for clarifying offline. You're 100% correct matching events should be hidden in property definition details. Made that fix

**Comment by liyiy** [comment]
@alexkim205 looks like something is bugging out?

https://user-images.githubusercontent.com/25164963/172261132-12489415-39ad-4fbc-881a-f0be77d08909.mov

**Review by liyiy — APPROVED** [review]
looks good after that edit button removal is in for non-licensed instances

**Pull request merged**